### DISCUSSION TITLE
Cannot Export .ipynb into .pdf file

### DISCUSSION THREAD

**Discussion by chocolatmint** [teamalgoritma/community#803] [Q&A] [answered]
Salam kenal,

Saya mau meng-export file .ipynb saya menjadi .pdf. Saat saya mencobanya, terdapat error bahwa nbconvert belum diinstall. Saya pun melakukan pip install nbconvert. Kemudian saya melakukan export pdf kembali, kali ini errornya berubah menjadi:
`nbconvert.utils.pandoc.PandocMissing: Pandoc wasn't found.`

Hal-hal yang sudah saya lakukan:
1. Saya sudah mencoba pip install pandoc, dan pada saat menggunakan conda list, lib pandoc sudah terinstall. Namun masih gagal pada saat export.
<img width="618" alt="image" src="https://github.com/teamalgoritma/community/assets/9330628/93c1886b-e7f3-499f-8e76-c909ea8d7484">

2. Saya sudah mencoba untuk close dan open VS Code tapi masih belum bisa export juga. 

3. Saya juga sudah mencoba untuk melakukan brew install pandoc, tapi saya mendapat error:
<img width="567" alt="image" src="https://github.com/teamalgoritma/community/assets/9330628/1acbfc0e-3af3-4852-8f19-0f60e3c2918c">

Mohon bantuannya agar saya bisa mengexport .ipynb menjadi .pdf.

Masalah terjadi di Macbook Pro, intel 2020 OS BigSur

**Comment by IrfanChairurrachman** [comment, marked as answer]
Dear @chocolatmint ,

Meng-export ipynb menjadi pdf membutuhkan step dan requirements tambahan karena tidak sesimpel meng-export menjadi html, dalam hal ini adalah nbconvert. Namun nbconvert juga membutuhkan requirementsnya tersendiri, antara lain _pandoc_, _playwright_ dan juga _mesin latex_ yang harus diinstall secara terpisah.

Step untuk mempersiapkan requirement nbconvert dapat diikuti melalui website berikut, Anda dapat menyesuaikan dengan operating system yang digunakan: [installing nbconvert](https://nbconvert.readthedocs.io/en/latest/install.html#installing-nbconvert)

Sekian jawaban dari kami. Bila ternyata informasi ini dapat menjawab pertanyaan Anda, Anda dapat “Mark as answer” jawaban ini.

**Comment by kinantytasyao** [comment]
Halo @chocolatmint , karena tidak ada konfirmasi lebih lanjut dan untuk menjaga agar forum diskusi tetap berjalan kondusif, kami izin melakukan lock/close diskusi ini. Semoga dapat dipahami.
